src/main/java/org/prebid/server/bidder/adnuntius/AdnuntiusBidder.java

```diff
@@ -85,24 +89,41 @@ public AdnuntiusBidder(String endpointUrl, Clock clock, JacksonMapper mapper) {
     public Result<List<HttpRequest<AdnuntiusRequest>>> makeHttpRequests(BidRequest request) {
         final Map<String, List<AdnuntiusAdUnit>> networkToAdUnits = new HashMap<>();
         boolean noCookies = false;
+        final ExtUser extUser;
+        final ObjectNode siteExtData;
 
-        for (Imp imp : request.getImp()) {
-            final ExtImpAdnuntius extImpAdnuntius;
-            try {
+        try {
+            extUser = extractUserExt(request.getUser());
+            siteExtData = extractData(request.getSite());
+            for (Imp imp : request.getImp()) {
+                final ExtImpAdnuntius extImpAdnuntius;
                 validateImp(imp);
                 extImpAdnuntius = parseImpExt(imp);
-            } catch (PreBidException e) {
-                return Result.withError(BidderError.badInput(e.getMessage()));
-            }
 
-            noCookies = resolveIsNoCookies(extImpAdnuntius);
-            final String network = resolveNetwork(extImpAdnuntius);
+                noCookies = resolveIsNoCookies(extImpAdnuntius);
+                final String network = resolveNetwork(extImpAdnuntius);
 
-            networkToAdUnits.computeIfAbsent(network, n -> new ArrayList<>())
-                    .add(makeAdnuntiusAdUnit(imp, extImpAdnuntius));
+                networkToAdUnits.computeIfAbsent(network, n -> new ArrayList<>())
+                        .add(makeAdnuntiusAdUnit(imp, extImpAdnuntius));
+            }
+        } catch (PreBidException e) {
+            return Result.withError(BidderError.badInput(e.getMessage()));
         }
 
-        return Result.withValues(createHttpRequests(networkToAdUnits, request, noCookies));
+        return Result.withValues(createHttpRequests(networkToAdUnits, request, noCookies, extUser, siteExtData));
+    }
+
+    private static ExtUser extractUserExt(User user) {
+        return Optional.ofNullable(user)
+                .map(User::getExt)
+                .orElseThrow(() -> new PreBidException("Failed to parse user.ext"));
+    }
```

Comment:
Actually, `user.ext` can be `null` according to the GO logic, so there is no need to throw this exception.